modules/op-disabling-pipeline-templates.adoc

```diff
@@ -0,0 +1,37 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-disabling-pipeline-templates_{context}"]
+= Disabling pipeline templates
+
+By default, the `TektonAddon` custom resource (CR) installs pipeline templates with {pipelines-shortname} on the cluster.
+
+You can disable the installation of pipeline templates by setting the parameter value to `false` in the `TektonConfig` custom resource (CR).
```

Comment:
By default, the `TektonAddon` custom resource (CR) installs `tasks` and `pipelineTemplates` resources along with {pipelines-shortname} on the cluster.

You can disable installation of the `tasks` and `pipelineTemplates` resources by setting the parameter value to `false` in the `.spec.addon` specification.

Reply:
But i need to try this configuration combination on a cluster

Reply:
Wait, do we document these pipeline templates anywhere? I don't recall them being available earlier. 

If we do not document pipeline templates for now, I would suggest maybe taking this module out for now and creating a separate story to document pipeline templates, outside the 1.17 time frame.

WDYT @savitaashture ?

Reply:
@mramendi we only have this disabling module AFAIK, but I would keep it at least for now, because we already had it in the previous doc versions. We can definitely build on it later though. 

But I can also comment it out, if you think it really should not be there  :+1: